src/teamforge.iced

```diff
@@ -247,6 +248,9 @@ class Team
       out.team_merkle[@id] =
         seqno: link.for_client.seqno
         link_id: link.link_id
+      out.team_merkle["#{@id}-seqno:#{link.for_client.seqno}"] =
+        seqno: link.for_client.seqno
+        link_id: link.link_id
```

Comment:
Could we throw in some explanatory comments about what the `team_merkle` structure is and who's consuming it? Not  sure where is the right place tho.